src/api/entities/AuditTrail.ts

```diff
@@ -10,16 +10,21 @@ export enum SharingAction {
 
 export enum AuditTrailEvents {
   UpdateSharingPermissions = 'UpdateSharingPermissions',
+  ApproveAccount = 'ApproveAccount',
 }
 
-export type AuditTrailEventData = UpdateSharingPermissionEventData;
+export type AuditTrailEventData = UpdateSharingPermissionEventData | ApproveAccountEventData;
 
 export type UpdateSharingPermissionEventData = {
   siteId: number;
   action: SharingAction;
   sharingPermissions: number[];
 };
 
+export type ApproveAccountEventData = {
+  siteId: number;
```

Comment:
From the parser on the API endpoint, it looks like a few things can be changed as well - name and types. I think it would be good to record any changes to those as well as the site ID being chosen.

Reply:
That's a good point. Should we record both? or just the original name and type? as the new name and type will be stored in the participant

Reply:
I'd probably prefer to see both what the values were, and the new ones - so if someone gets approved with changed details and there's a query raised about it, we can find out how it was changed and what it was originally.

Reply:
Name and typeIds are now added to the event data